esp/mlrs-wifi-bt-bridge/mlrs-wifi-bt-bridge.ino

```diff
@@ -213,9 +241,37 @@ void loop()
 
     //-- here comes the core code, handle wifi connection and do the bridge
 
-    uint8_t buf[256]; // working buffer
+    const int bufsize = 256;
+    uint8_t buf[bufsize]; // working buffer
 
-#if WIFI_PROTOCOL == 1 // UDP
+#if (WIFI_nBLUETOOTH == 0) // Bluetooth
```

Comment:
could you pl move the BT to after the udp/tcp stuff?

Reply:
My reasoning was that option 0 comes before option 1. No?

Reply:
Fixed in latest commit, as now using:
0: WiFi TCP
1: WiFi UDP
2: Bluetooth